index.html

```diff
@@ -118,20 +118,20 @@
         <dl>
           <dt><dfn>scrollTop</dfn></dt>
           <dd>
-            Returns the distance from the top edge of the visual viewport to the top edge of the layout viewport.
+            Returns the location of the top edge of the visual viewport relative to the top edge of the layout viewport in CSS pixels.
           </dd>
           <dt><dfn>scrollLeft</dfn></dt>
           <dd>
-            Returns the distance from the left edge of the visual viewport to the left edge of the layout viewport.
+            Returns the location of the left edge of the visual viewport relative to the left edge of the layout viewport in CSS pixels.
           </dd>
 
           <dt><dfn>pageX</dfn></dt>
           <dd>
-            Returns the location of the left edge of the visual viewport from the left edge of the document.
+            Returns the x-coordinate, relative to the <a href="https://drafts.csswg.org/css-display-3/#initial-containing-block">initial containing block</a> origin, of the left of the visual viewport in CSS pixels.
```

Comment:
"of the left of the visual viewport" sounds odd. Maybe "of the left edge of the visual viewport"?